test/helpers.ts

```diff
@@ -200,3 +200,7 @@ export function defaultRegion(overrideList?: string[]): string {
   ];
   return regionList[Math.floor(Math.random() * regionList.length)];
 }
+
+export function itDocs(...args: any[]): any {
+  return (it as unknown as (...args: any[]) => {})(...args);
```

Comment:
Maybe this solve then issue of the `undefined`s around on the `done` functions?
```suggestion
  return (it as unknown as (...args?: any[]) => {})(...args);
```
(just thinking out loud)

Reply:
no, i tried already... A rest parameter cannot be optional.ts(1047)